packages/account-web-vault/src/web-user.ts

```diff
@@ -0,0 +1,250 @@
+import { Network, Context, Profile } from '@verida/client-ts';
+import { VaultAccount, hasSession, VaultAccountConfig } from './index';
+import { EventEmitter } from 'events'
+import { MessageSendConfig } from '@verida/client-ts/dist/context/interfaces';
+
+const VeridaEvents = new EventEmitter()
+
+export interface WebUserProfile {
+    name?: string
+    avatarUri: string
+    country?: string
+    description?: string
+}
+
+export interface WebUserConfig {
+    accountConfig: VaultAccountConfig
+    clientConfig: ClientConfig
+    contextConfig: ContextConfig
+    networkConfig: NetworkConnectionConfig
+}
+
+export interface WebUserMessage {
+    subject: string
+    text: string
+    link?: WebUserMessageLink
+}
+
+export interface WebUserMessageLink {
+    url: string
+    text: string
+}
+
+/**
```

Comment:
Worth adding jsdoc mentioning the usage: start by `WebUser.configure(...)`, then `WebUser.connect()`, ...

Reply:
Good point